newsletter-september-2023.rst

```diff
@@ -0,0 +1,76 @@
+.. post:: September 5, 2023
+   :tags: newsletter, python
+   :author: Anthony
+   :location: GEG
+   :category: Newsletter
+
+Read the Docs newsletter - September 2023
+=========================================
+
+News and updates
+----------------
+
+- 🚀 :doc:`We started testing a new flyout menu </addons-flyout-menu-beta>`
+  as part of our beta test for documentation addons.
+  The beta test is currently limited to projects using the ``build.commands``
+  configuration key.
+  
+- 🛣️ We continue to have a number deprecations in progress.
+  We announced this month deprecations of :doc:`installing using system packages </drop-support-system-packages>`,
+  :doc:`the configuration key build.image </use-build-os-config>`,
+  and :doc:`installation of pinned versions of Sphinx and MkDocs </defaulting-latest-build-tools>`.
+  Keep an eye on your email for any deprecation notifications,
+  as we will continue to notify maintainers of projects that might be impacted.
+
+- 📚 The Read the Docs Sphinx theme package, ``sphinx-rtd-theme``, had two releases.
+  Version 1.3.0 was released, adding support for Sphinx 7.0.
+  Version 2.0.0rc2 is also now out.
+  This is a release candidate that will remove support for HTML4 output and will drop support for Sphinx versions prior to 5.0.
+  We will be announcing the release candidate more this month and will be looking to have feedback from users.
+
+- 🔐 A security advisory involving symlink abuse during project builds was raised and patched.
```

Comment:
Maybe link to the advisory?

Reply:
Good call, we've linked to advisories at least once in the past, so I'll do the same.